wolfcrypt/src/sm3.c

```diff
@@ -0,0 +1,7 @@
+
+#ifdef WOLFSSL_SM3
```

Comment:
Please include the wolfssl/wolfcrypt/settings.h for these empty files.

Reply:
fixed